src/Integration.UnitTests/Suppression/SuppressedIssueMatcherTests.cs

```diff
@@ -130,20 +145,25 @@ private IFilterableIssue CreateIssueToMatch(string ruleId, int? startLine, strin
                 FilePath = "well known file path"
             };
 
-        private SonarQubeIssue CreateServerIssue(string ruleId, int? startLine, string lineHash)
-            => new SonarQubeIssue(null, null, lineHash, null, null, ruleId, false, SonarQubeIssueSeverity.Info,
-                DateTimeOffset.MinValue, DateTimeOffset.MinValue,
-                startLine.HasValue
-                    ? new IssueTextRange(startLine.Value, 1, 1, 1)
-                    : null,
-                flows: null);
+        private SonarQubeIssue CreateServerIssue(string ruleId, int? startLine, string lineHash, bool isSuppressed)
+        {
+            var sonarQubeIssue = new SonarQubeIssue(null, null, lineHash, null, null, ruleId, false, SonarQubeIssueSeverity.Info,
+                 DateTimeOffset.MinValue, DateTimeOffset.MinValue,
+                 startLine.HasValue
+                     ? new IssueTextRange(startLine.Value, 1, 1, 1)
+                     : null,
+                 flows: null);
+
+            sonarQubeIssue.IsResolved = isSuppressed;
+
+            return sonarQubeIssue;
+        }
 
         private void ConfigureServerIssues(
```

Comment:
Removed issueToMatch as we no longer needed it (The new Get() does not have any parameters)